runtime-testsuite/test/org/antlr/v4/test/runtime/go/BaseGoTest.java

```diff
@@ -80,10 +80,36 @@ private static void cacheGoRuntime(File tmpPackageDir) throws Exception {
 		}
 	}
 
+    private static void setupGoMod() throws Exception{
```

Comment:
Are you sure we have to do this every time and not only once inside `cacheGoRuntime`?

Reply:
True, I'll refactor it